kernel/declarations.ml

```diff
@@ -182,21 +182,34 @@ type regular_inductive_arity = {
 
 type inductive_arity = (regular_inductive_arity, template_arity) declaration_arity
 
+(** {7 Datas specific to a single type of a block of mutually inductive type } *)
 type one_inductive_body = {
+
 (** {8 Primitive datas } *)
 
-    mind_typename : Id.t; (** Name of the type: [Ii] *)
+    mind_typename : Id.t; (** Name of the type: [Iᵢ] *)
 
-    mind_arity_ctxt : Constr.rel_context; (** Arity context of [Ii] with parameters: [forall params, Ui] *)
+    mind_arity_ctxt : Constr.rel_context;
+ (** Arity context of [Iᵢ]. It includes the context of parameters,
+     that is, it has the form [paramdecls, realdeclsᵢ] such that [Uᵢ]
+     (see above) is [forall realdeclsᵢ, sᵢ] for some sort [sᵢ] and
+     such that [Iᵢ] has thus type [forall paramdecls, forall
+     realdeclsᵢ, sᵢ]. The context itself is represented internally as
+     a list in reverse order
+     [[realdeclᵢrᵢ;...;realdeclᵢ₁;paramdeclₘ;...;paramdecl₁]]. *)
```

Comment:
```suggestion
     [[realdeclᵢⱼ;...;realdeclᵢ₁;paramdeclₘ;...;paramdecl₁]]. *)
```

Reply:
(Or maybe some other subscript if we want `j` to be for constructors)

Reply:
You mean that the excess of precision eventually makes things more confusing?

Reply:
You mean `r` is supposed to be a subscript? I thought it was a typo
ie I guess you intended `realdecl_{i,r_i}` but I initially read `realdecl_{i,i}` with a nonsense `r` added inside

Reply:
Exactly. And I agree this is confusing.